docs/xrunjdwp.md

```diff
@@ -0,0 +1,112 @@
+﻿<!--
+* Copyright (c) 2017, 2024 IBM Corp. and others
+*
+* This program and the accompanying materials are made
+* available under the terms of the Eclipse Public License 2.0
+* which accompanies this distribution and is available at
+* https://www.eclipse.org/legal/epl-2.0/ or the Apache
+* License, Version 2.0 which accompanies this distribution and
+* is available at https://www.apache.org/licenses/LICENSE-2.0.
+*
+* This Source Code may also be made available under the
+* following Secondary Licenses when the conditions for such
+* availability set forth in the Eclipse Public License, v. 2.0
+* are satisfied: GNU General Public License, version 2 with
+* the GNU Classpath Exception [1] and GNU General Public
+* License, version 2 with the OpenJDK Assembly Exception [2].
+*
+* [1] https://www.gnu.org/software/classpath/license.html
+* [2] https://openjdk.org/legal/assembly-exception.html
+*
+* SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0-only WITH Classpath-exception-2.0 OR GPL-2.0-only WITH OpenJDK-assembly-exception-1.0
+-->
+
+# -Xrunjdwp
+
+The `-Xrunjdwp` option enables the Java&trade; Debug Wire Protocol (JDWP) agent in the VM. The JDWP agent provides a connection point for the debugger to debug a Java application in that VM.
+
+:fontawesome-solid-pencil:{: .note aria-hidden="true"} **Notes:**
+
+- You can enable JDWP support on Checkpoint/Restore In Userspace (CRIU) restore by specifying either the `-Xrunjdwp` or the `-agentlib:jdwp` command-line options pre-checkpoint.
+- Instead of specifying `-Xrunjdwp` or `-agentlib:jdwp` as command-line options pre-checkpoint when you restore a checkpoint image, you can add these command-line options by using the `CRIUSupport.registerRestoreOptionsFile` API.
```

Comment:
```suggestion
- Instead of specifying `-Xrunjdwp` or `-agentlib:jdwp` as command-line options pre-checkpoint when you restore a checkpoint image, you can add these command-line options into a restore option file and specify it via the `CRIUSupport.registerRestoreOptionsFile` API.
```